WireAPI/Tests/WireAPITests/UpdateEvent/TeamEventDecodingTests.swift

```diff
@@ -69,7 +69,12 @@ final class TeamEventDecodingTests: XCTestCase {
 
     private enum Scaffolding {
 
+        static func date(from string: String) -> Date {
+            ISO8601DateFormatter.fractionalInternetDateTime.date(from: string)!
```

Comment:
Some endpoints return the date without fractional seconds.
I'm not sure, but I would consider it even possible that an endpoint returned the one format in earlier versions, and since I raised the issue it might have been adjusted, so that it returns a different format in later versions.
I'm just assuming.
Have you investigated that?

Reply:
yes, event payloads are not versioned like "regular" API endpoints (v0, v1, v2 etc), I also double checked on Android as well. Also I've checked with Android team, this is property is not optional on their side.

The event payload model response is as follows (e.g `conversation.otr-asset-add`) 

```
{
      "type": "conversation.otr-asset-add",
      "time": "2015-08-19T15:32:28.975Z",
      "data": { ... },
      "qualified_conversation": {
            "domain": "staging.zinfra.io",
            "id": "e16babfa-308b-414e-b6e0-c59517f723db"
          },
      "qualified_from": {
            "domain": "staging.zinfra.io",
            "id": "76ebeb16-a849-4be4-84a7-157654b492cf"
          }
}
```